aksk/pkg/iam/driver/keystone/keystone.go

```diff
@@ -220,3 +244,58 @@ func (iam *KeystoneIam) DownloadAkSk(ctx context.Context, in *pb.GetAkSkRequest)
 	json.Unmarshal(bodyBytes, &akskListout)
 	return akskListout, nil
 }
+
+func isValidUserId(userId string, token string, host string) bool {
```

Comment:
IMHO, The method can be receiver method of KeystoneIAM as it is not exposed outside of package and also using client object of KeystoneIAM. The statement might not be required then.
Eventually client can be used as iam.client.Do and host argument may not be required

Reply:
fixed